app/models/miq_request_workflow.rb

```diff
@@ -1328,8 +1328,12 @@ def get_pxe_server
     PxeServer.find_by(:id => get_value(@values[:pxe_server_id]))
   end
 
+  def get_template
```

Comment:
Use the `get_source_vm` method instead.  Note that in the case of service catalog items this can return a `nil` value.